server/lib/nl/fulfillment/base.py

```diff
@@ -249,3 +253,38 @@ def _svg_or_topic_to_svs(state: PopulateState, sv: str,
     return charts
 
   return []
+
+
+def open_topics_ordered(svs: List[str]) -> List[str]:
+  opened_svs = []
+  sv_set = set()
+  for rank, sv in enumerate(svs):
+    for sv in _open_topic_in_var(sv, rank):
```

Comment:
both the inner and outer loops are using the same var name `sv`?

Reply:
good catch!